portfolio/styles.css

```diff
@@ -0,0 +1,185 @@
+html {
+    scroll-behavior: smooth;
+}
```

Comment:
Amazing work with the styling. You've got a solid grip on Flexbox in your stylesheet. Your use of display: flex; to create responsive and organized containers is spot-on, showing your understanding of flexible design principles. Keep up the good work!